src/test/Makefile-client.am

```diff
@@ -234,7 +235,7 @@ ceph_test_rados_api_aio_CXXFLAGS = $(UNITTEST_CXXFLAGS)
 bin_DEBUGPROGRAMS += ceph_test_rados_api_aio
 
 ceph_test_rados_api_list_SOURCES = test/librados/list.cc
-ceph_test_rados_api_list_LDADD = $(LIBRADOS) $(UNITTEST_LDADD) $(RADOS_TEST_LDADD)
+ceph_test_rados_api_list_LDADD = $(LIBRADOS) $(UNITTEST_LDADD) $(RADOS_TEST_LDADD) $(CEPH_GLOBAL)
```

Comment:
@wjwithagen i don't quite understand this change, why we need $(CEPH_GLOBAL) here?

as ceph_test_rados_api_list is a typical librados client, in theory, it should only rely on librados.